index.html

```diff
@@ -231,5 +231,58 @@ <h2>About me</h2>
         </ul>
       </div>
     </section>
+    <!-- contact page -->
+    <section class="contact-page">
```

Comment:
[optional] I opened the mobile version and as you can see the site has horizontal scrolling, which makes it difficult to fill the form on a mobile device , please try to fix the horizontal scrolling please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/200627469-6db00839-0932-4239-b17d-e6b5b91b6a5e.png)
![image](https://user-images.githubusercontent.com/97521557/200627525-3bbdc697-9360-4fda-888f-2f49e4343de2.png)